ctest-next/templates/test.c

```diff
@@ -0,0 +1,21 @@
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+
+{% for header in headers %}
+    #include <{{ header }}>
+{% endfor %}
```

Comment:
```suggestion
{% for header in headers %}
#include <{{ header }}>
{% endfor %}
```

I think the templates probably shouldn't indent within `{%...%}` directives. Slightly harder to read the template, but the output looks much more sane.

Ideally we'd pass the output through `rustfmt` / `clang-tidy`, but we can't assume those are available.